### ISSUE TITLE
List of User/Supporters

### ISSUE THREAD

**Issue by curran** [share/sharedb#182]
It may be useful to have a listing of companies and projects using ShareDB, in the README somewhere.

I know there are many people using ShareDB in production apps today, it's just not clear to newcomers.

We can improve the initial impression of ShareDB by listing users and supporters.

Here are a few that come to mind:

 * [Lever](https://www.lever.co/)
 * [Retrium](https://www.retrium.com/) (not 100% sure, but I think so based on vague memories)
 * [Decision Mapper](http://decisionmapper.com/) (at least they used to use Derby heavily, not sure about now)
 * [Datavis.tech](https://datavis.tech/curran) (my company)

Any others? Who is out there using ShareDB and interested in its long-term health as an OSS project? Please post here!

**Comment by houshuang** [comment]
ShareDB is crucial to the way we design our collaborative learning authoring/workflow tool FROG github.com/chili-epfl/FROG. I wrote more about how we use ShareDB here https://groups.google.com/forum/#!msg/sharejs/N7QBY-qI2O4/EhFPxF2GAgAJ.

We are very early-stage, but have already been used with multiple lectures of 300 students this fall. Also currently working on automatically detecting collaborative editing behaviour of students - in-progress project here https://github.com/adrian-pace/FROG-analytics

Very happy to talk to others about collaboration, sharing best practices, scaling etc.

**Comment by zag2art** [comment]
Yup, we still use Derby in a few projects even though we also use Sharedb+React in the others (Derby was not accepted by the client).

**Comment by DetweilerRyan** [comment]
Not only do we use ShareDb in production at Retrium but it is considered a fundamental part of our tech stack. Therefore, we'd be happy to have Retrium listed as a supporter/user of sharedb.

**Comment by curran** [comment]
@DetweilerRyan Nice!

**Comment by curran** [comment]
Today I learned https://reedsy.com is using ShareDB for editing books! So cool.. via https://github.com/share/sharedb/issues/223 /cc @alecgibson
